src/js/actions/type.js

```diff
@@ -132,15 +132,24 @@ define(function (require, exports) {
             setFacePromise = locking.playWithLockOverride(document, layers, setFacePlayObject, typeOptions),
             updatePromise = this.transfer(updatePostScript, document, layers, postscript, family, style);
 
-        return Promise.join(updatePromise, setFacePromise).bind(this).then(function () {
-            if (!modal) {
-                return this.transfer(layerActions.resetBounds, document, layers);
-            }
-        });
+        return Promise.join(updatePromise, setFacePromise)
+            .bind(this)
+            .then(function () {
+                if (!modal) {
+                    var anylayerTextWarning = layers.some(function (layer) {
+                                return layer.textWarningLevel > 0;
+                            });
+                    if (anylayerTextWarning) {
+                        return this.transfer(layerActions.resetLayers, document, layers, true);
+                    } else {
+                        return this.transfer(layerActions.resetBounds, document, layers);
+                    }
+                }
+            });
     };
     setPostScript.reads = [locks.JS_DOC];
     setPostScript.writes = [locks.PS_DOC, locks.JS_UI];
-    setPostScript.transfers = [updatePostScript, layerActions.resetBounds];
+    setPostScript.transfers = [updatePostScript, layerActions.resetBounds,layerActions.resetLayers];
```

Comment:
Because we don't have `requireSpaceAfterComma`, which is a new feature to JSCS 6 days agao.
https://github.com/jscs-dev/node-jscs/blob/94f175eec822f62528e6e5ca5aab0eb1de037243/lib/rules/require-space-after-comma.js